org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/DictionaryTestCase.xtend

```diff
@@ -0,0 +1,231 @@
+package org.uqbar.project.wollok.tests.interpreter
+
+import org.junit.Test
+import org.uqbar.project.wollok.interpreter.core.WollokProgramExceptionWrapper
+
+/**
+ * @author jfernandes
+ */
+class DictionaryTestCase extends AbstractWollokInterpreterTestCase {
+	
+	def instantiateObjectsDictionary() {
+		"val dictionary = {foo:1, bar:'hello', baz: {:}}"
```

Comment:
I had to add it in order to avoid ambiguities with the expression block. Otherwise the following would be ambiguous:
```
if (true) 
  {}
```

Ruby doesn't have that issue since they don't use braces. And Groovy adopted the same solution (an empty dictionary in Groovy is `[:]`)

But I am open to alternatives here, Ink it can be done better. What do you suggest?